components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -950,14 +950,14 @@ SUBROUTINE p3_main_main_loop(kts, kte, kbot, ktop, kdir, log_predictNc, dt, odt,
 
  END SUBROUTINE p3_main_main_loop
 
- subroutine p3_main_post_loop(kts, kte, kbot, ktop, kdir, &
+ subroutine p3_main_post_main_loop(kts, kte, kbot, ktop, kdir, &
```

Comment:
Is this being renamed so as not to conflict with another subroutine of the same name?

Reply:
No, just to be consistent with the other pieces of p3_main that were split off.